megatron/optimizer/distrib_optimizer.py

```diff
@@ -237,11 +237,13 @@ def build_optimizer_group_ranges(cls, param_groups, model_gbuf_ranges):
                     local_param_group_map[param] = \
                         (group_index, len(group_range["params"]) - 1)
 
-        # Squeeze zero-size group ranges.
         for group_index, group_range in enumerate(group_ranges):
             group_range["orig_group"] = param_groups[group_index]
-            group_range["orig_group_idx"] = param_groups[group_index]
```

Comment:
Why did you change this assignment to `group_index` from `param_groups[group_index]`?

Reply:
I think `group_range["orig_group_idx"]` aims to store the original group index as the name suggests. However, `group_range["orig_group_idx"] = param_groups[group_index]` stores the original param group again just as the above`group_range["orig_group"] = param_groups[group_index]` does. Besides, I cannot find any usages of `group_range["orig_group_idx"]` in the code base for now. It's also OK to revert this back.